server/src/utils/WorkspaceFileRetriever.ts

```diff
@@ -14,6 +14,7 @@ export class WorkspaceFileRetriever {
       ignore,
       followSymbolicLinks: false,
       suppressErrors: true,
+      dot: true,
```

Comment:
Are there any performance implications for including `.` files for our other uses?

Reply:
I tested our indexing benchmark on our test monorepos and the performance is not impacted by enabling this flag.